config/application.rb

```diff
@@ -8,7 +8,8 @@
 
 module Hydranorth
   class Application < Rails::Application
-
+    config.ezid_url = "http://hydranorthdev.library.ualberta.ca/files/"
```

Comment:
Does this need to be hard coded? ezid_url is only used in one place.  Could this be replaced by ```Sufia::Engine.routes.url_helpers.generic_file_url```?